source/enable-autocomplete.txt

```diff
@@ -0,0 +1,94 @@
+.. _vs-code-enable-autocomplete:
+
+===================
+Enable Autocomplete
```

Comment:
This is not exactly correct. The autocomplete in playgrounds is enabled by default. The only exception is autocomplete within strings and vscode snippets. Those are turned off not by our extension, but by vscode itself for better performance. I am not sure if we want to recommend users to always configure the `editor.quickSuggestions` setting because it will affect not only MongoDB playgrounds but all files that users open in vscode, which can affect the performance of vscode. We can explain to them how to do it (if they often require autocomplete in strings), but also warn them about the consequences. See how we described it in our extension readme https://github.com/mongodb-js/vscode?tab=readme-ov-file#autocomplete

Reply:
I added a disclaimer about this in the about this task section. I also clarified the option of use the keyboard shortcut instead, in the same section instead of at the end of the task.

Reply:
Would it be more clear for readers if we reflect this I the title as well? Currently, it makes an impression that we need to enable autocomplete for the whole extension.

Reply:
I updated the page title to `Enable String Literal Autocomplete` to reflect this.